src/test/scala/de/bripkens/ha/reporting/ConsoleReporterSpec.scala

```diff
@@ -30,5 +30,13 @@ class ConsoleReporterSpec extends BaseAkkaSpec {
       }
 
     }
+
+    "log a warning when it receives an unknown message" in {
```

Comment:
This is just to show, that the logging works. I think we should remove this tests, since it just makes sure, that akke works as expected. If you want to inspect the actual log message, just remote the wrapping EventFilter intercept code. You can then inspect the log message in the console.